style.css

```diff
@@ -1,16 +1,17 @@
 /*
 Theme Name: UnderStrap
 Theme URI: https://understrap.com
-Author: Holger Koenemann
-Author URI: https://www.holgerkoenemann.de
+Author: the Understrap Authors
```

Comment:
I think we should either stick to UnderStrap or Understrap. Which one is the "correct" one according to the new owners?

Reply:
We are moving to Understrap with a lowercase "s" - but I am fine with slowly migrating past references that include the uppercase S, it is not a huge deal in my mind.

I would also like to reference Howard Development & Consulting in the Author line in some way since that is also how it appears in the directory now - but I am open to everyone's input on the best way to provide the proper credit across the board.

Reply:
`Author: the Understrap Authors and Howard Development & Consulting`?